frontend/src/components/Navbar.tsx

```diff
@@ -1,11 +1,38 @@
 import './Navbar.css';
-import logo from '../images/maid-logo.png';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
+import {faUser} from '@fortawesome/free-solid-svg-icons'
+import {Dispatch, SetStateAction} from "react";
+
+
+type NavbarProps = {
+    me: string
+    handleLogout: () => void
+    setProfile: Dispatch<SetStateAction<boolean>>
+}
+
+export function Navbar (props: NavbarProps) {
+
+    const showProfile = () => {
+        props.setProfile(true)
+    }
 
-export function Navbar () {
     return (
-        <div className={"navbar-container"}>
-            <img className={"logo"} src={logo} alt={"Logo"}/>
-            <p>dev_MAID v0.1</p>
-        </div>
+        <label className="navbar-container">
```

Comment:
Rein theoretisch könntest du das ganze auch in einen <nav></nav> wrappen... Ist ein eigener Tag für eine NavBar... Ist nichts Weltbewegendes, nur für die html semantik...